docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -529,18 +529,36 @@ const unsubscribe = room.subscribe("connection", (status) => {
 
 ### Room.subscribe("error") [#Room.subscribe.error]
 
-Subscribe to potential room connection errors.
+Subscribe to unrecoverable room connection errors. This event will be emitted
+right before the client disconnects and won’t try reconnecting again.
 
 Returns an unsubscribe function.
 
 ```ts
 const unsubscribe = room.subscribe("error", (error) => {
-  if (error.code === 4005) {
-    // Maximum concurrent connections per room exceeded.
+  switch (error.code) {
```

Comment:
I'm considering typing error as it's always manually thrown by us to use an enum to avoid the switch with random numbers, what do you think? No need to do that right now

Reply:
Maybe. Would have to think about it. I wouldn't want to advertise the `WebsocketCloseCodes` enum publicly. Maybe a different one? Not doing it now anyway, let's punt on this for later.